experimental/sql/src/ssa_to_impl.py

```diff
@@ -148,6 +153,26 @@ def match_and_rewrite(self, op: RelSSA.Project, rewriter: PatternRewriter):
                                          predicates))
 
 
+@dataclass
+class InnerJoinRewriter(RelSSARewriter):
+
+  @op_type_rewrite_pattern
+  def match_and_rewrite(self, op: RelSSA.InnerJoin, rewriter: PatternRewriter):
+    predicates = Region.from_block_list([
+        Block.from_arg_types([
+            self.create_tuple_of_bag(op.left.typ),
+            self.create_tuple_of_bag(op.right.typ)
+        ])
+    ])
+    # The following loop moves the operations of (the old) op.predicates to the
+    # predicates region of the new operation.
```

Comment:
Isn't there a function in xdsl that does that? Can't there be more than one block?

Reply:
There isn't yet. There is only the option to move to a new region without block arguments. However, I feel that this is something we should add at some point.

Reply:
There cannot be more blocks in our current design. Clear this is an assumption on the IR, on which there is no verifier atm.